src/map_object_move.c

```diff
@@ -933,40 +934,40 @@ int sub_02064390 (MapObject * mapObj)
     if (sub_02062FDC(mapObj) == 0) {
         int v2 = MapObject_XPosPrev(mapObj);
         int z = MapObject_ZPosPrev(mapObj);
-        FieldSystem * v4 = MapObject_FieldSystem(mapObj);
+        FieldSystem * fieldSystem = MapObject_FieldSystem(mapObj);
 
-        v0 = sub_02054F94(v4, v2, z);
+        v0 = sub_02054F94(fieldSystem, v2, z);
         v2 = MapObject_GetXPos(mapObj);
         z = MapObject_GetZPos(mapObj);
-        v1 = sub_02054F94(v4, v2, z);
+        v1 = sub_02054F94(fieldSystem, v2, z);
     }
 
     sub_02062BF0(mapObj, v0);
     sub_02062BE0(mapObj, v1);
 
     if (sub_0205DF8C(v1) == 1) {
-        MapObject_SetStatusFlagOn(mapObj, (1 << 11));
+        MapObject_SetStatusFlagOn(mapObj, MAP_OBJ_STATUS_11);
         return 0;
     }
 
-    MapObject_SetStatusFlagOff(mapObj, (1 << 11));
+    MapObject_SetStatusFlagOff(mapObj, MAP_OBJ_STATUS_11);
     return 1;
 }
 
-void sub_02064418 (int param0, VecFx32 * param1, fx32 param2)
+void VecFx32_AddValInDirection (int dir, VecFx32 * vec, fx32 val)
```

Comment:
I think this one deserves whatever suffix we decide on for the similar `MapObject` function above.